### PR TITLE
SAK-41812 - Conversion scripts for the profile name pronunciation feature

### PR DESCRIPTION
**Pull request by mpellicer**
https://jira.sakaiproject.org/browse/SAK-41812

This PR is related to https://github.com/sakaiproject/sakai/pull/8006

### CODE REVIEW

File: docs/conversion/sakai_20_mysql_conversion.sql [hunk 1]
```diff
@@ -8,4 +8,8 @@ UPDATE CM_OFFICIAL_INSTRUCTORS_T SET INSTRUCTOR_ID = LOWER(INSTRUCTOR_ID);
 
 ALTER TABLE POLL_OPTION ADD OPTION_ORDER INTEGER;
 
--- END SAK-41391
\ No newline at end of file
+-- END SAK-41391
+
+-- START SAK-41812
+ALTER TABLE SAKAI_PERSON_T ADD PHONETIC_PRONUNCIATION LONGTEXT DEFAULT NULL;
```
Comment (ottenhoff) [inline docs/conversion/sakai_20_mysql_conversion.sql]: Why not just a simple varchar(255)?

Comment (mpellicer) [reply docs/conversion/sakai_20_mysql_conversion.sql]: Looks like @josecebe followed the same pattern of the HBM file here:
https://github.com/sakaiproject/sakai/blob/master/common/impl/src/java/org/sakaiproject/component/common/edu/person/SakaiPersonImpl.hbm.xml#L136

Defined as materialized_clob and Hibernate generates a LONGTEXT field.

Our client defined this field as varchar in oracle and failed, they needed to set a CLOB type.

Do you think it's worth changing the type in the HBM and the scripts?

File: docs/conversion/sakai_20_oracle_conversion.sql [hunk 2]
```diff
@@ -86,3 +86,8 @@ ALTER TABLE SAM_PUBLISHEDACCESSCONTROL_T ADD FEEDBACKSCORETHRESHOLD FLOAT(126);
 
 -- SAK-42498
 ALTER TABLE BULLHORN_ALERTS DROP COLUMN ALERT_TYPE;
+
+-- START SAK-41812
+ALTER TABLE sakai_person_t ADD phonetic_pronunciation longtext default null;
```
Comment (bjones86) [inline docs/conversion/sakai_20_oracle_conversion.sql]: `longtext` is not a valid Oracle data type as far as I'm aware.

Comment (mpellicer) [reply docs/conversion/sakai_20_oracle_conversion.sql]: Exactly, Hibernate is creating a clob instead:
image

**Comment by fostersdesign** [comment]
We discussed this on the Sakai Core team call today and decided that column size is too big. @mpellicer is going to create a new Jira issue to change the size in the code and in the database scripts.

**Pull request merged**